proposals/lip-0069.md

```diff
@@ -178,3 +178,7 @@ For example, the `seedReveal` property is added to the block asset in this hook
 This LIP is informational. It does not introduce any protocol change.
 
 [lip-0065]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0065.md
+
+## Reference Implementation
+
+[Update lisk-chain to support the on-chain architecture #5609](https://github.com/LiskHQ/lisk-sdk/issues/5609)
```

Comment:
I think, this one is slightly more correct: https://github.com/LiskHQ/lisk-sdk/issues/5603.